src/order/models.py

```diff
@@ -34,7 +34,7 @@
 ORDER_STATUS_ORDERED = ORDER_STATUS[0][0]
 
 SIZE_CHOICES = (
-    ('', _('Serving size')),
+    ('', ''),
```

Comment:
Hi @lingxiaoyang, I think this change implies a migration that you don't have added in your PR. Could you check this please?